package.json

```diff
@@ -32,7 +33,7 @@
     "install:node-v8": "ts-node ./scripts/install-node-v8.ts",
     "build": "tsc -p ./src && tsc lib/tc.ts",
     "watch-build": "tsc -w -p ./src && tsc -w -p ./lib",
-    "test": "./node-v8/bin/node ./node_modules/jest/bin/jest.js ",
+    "test": "bash scripts/test",
```

Comment:
That's not true `yarn test` should work in all platforms. It's working on my windows machine and appveyor https://ci.appveyor.com/project/nidin/turboscript/build/1.0.8